### PR TITLE
gec: formatting

### PR DESCRIPTION
**Pull request by caseyfitz**
* Update readme to link documentation and provide instructions for running local steps
* Add optional use of .env in config
* formatting

### CODE REVIEW

File: grammar-error-corrector/.env.example
Comment (caseyfitz) [reply grammar-error-corrector/.env.example:2]: Haha @shaffeeullah 🙏  this obviously wasn't supposed to come through (that was from our demo!). I like the notebook changes too. if you want to pull these in you can just rebase after merging your updates and ignore my notebook changes. Your call.

**Review by shaffeeullah — APPROVED** [review]

**Pull request merged**
